frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -1519,48 +1539,3 @@ export const getCurrentPlayerTime = (logicProps: SessionRecordingPlayerLogicProp
     const playerTime = sessionRecordingPlayerLogic.findMounted(logicProps)?.values.currentPlayerTime || 0
     return Math.floor(playerTime / 1000)
 }
-
-export const manageConsoleWarns = (cache: any, onIncrement: (count: number) => void): (() => void) => {
```

Comment:
this stopped us printing to the console
but didn't stop the call to console
so we've moved it to a logger we can pass to rrweb (which supports that now) which doesn't call console until the debounce timer passes